l10cloud-formation/lesson10ap.yml

```
@@ -0,0 +1,117 @@
+AWSTemplateFormatVersion: 2010-09-09
+Resources: 
+#EC2の作成
+  lesson10ec2:
+    Type: AWS::EC2::Instance
+    Properties:
+      KeyName: lesson5
+      DisableApiTermination: false
+      ImageId: ami-04f0680f68f076681
+      InstanceType: t2.micro
+      Monitoring: true
+      NetworkInterfaces: 
+        - AssociatePublicIpAddress: "true"
+          DeviceIndex: "0"
+          GroupSet: 
+            - !ImportValue l10ec2securityoutName
```

命名規則がケバブケース、ローワーキャメルケース、パスカルケースどれなのか読み取れなかったので、れにするかは決めた方が良いかなと思います！
例えばローワーキャメルケースだとこう表現します！
```suggestion
            - !ImportValue l10Ec2SecurityOutName
```

---

私が指摘した部分以外も関係する話ですので、命名規則を決定したら規則に則って全て修正ください！

---

命名規則をローワーキャメルケースに統一しました。ただし、s3は大文字を使えないため、小文字のままです。